source/common/upstream/cluster_manager_impl.cc

```diff
@@ -971,6 +971,11 @@ void ClusterManagerImpl::drainConnections() {
   });
 }
 
+bool ClusterManagerImpl::checkActiveStaticCluster(const std::string& cluster) {
```

Comment:
Do you have a unit test for this method?

Reply:
noted.